goreadme.go

```diff
@@ -142,6 +142,8 @@ type Config struct {
 	// ImportPath is used to override the import path. For example: github.com/user/project,
 	// github.com/user/project/package or github.com/user/project/version.
 	ImportPath string `json:"import_path"`
+	// GoDocURL is the Go Doc URL used in the GoDoc Badge.
+	GoDocURL string `json:"go_doc_url"`
```

Comment:
Maybe `godoc_url`, to be consistent with `godoc-url` in `main.go`?

Reply:
Indeed!